kore/src/Kore/Step/Simplification/Predicate.hs

```diff
@@ -121,10 +144,19 @@ simplify sideCondition =
                 NotF notF -> simplifyNot =<< traverse worker notF
                 ImpliesF impliesF -> simplifyImplies =<< traverse worker impliesF
                 IffF iffF -> simplifyIff =<< traverse worker iffF
+                CeilF ceilF ->
```

Comment:
Should we have a symmetric case for Floor here as well or is this fine because we're planning to get rid of Floor anyway?

Reply:
We should de-sugar `Floor` to `Not Ceil Not`, but we don't use `Floor` in practice, so I thought it wasn't important for getting this merged.